README.md

```diff
@@ -602,6 +602,12 @@ $driver = new Driver_Phantomjs($connection);
 
 $page = new Page();
 ```
+By default Phanatomjs have to be in your PATH, if you want you can set custom path.
+If file not found, will throw Exception.
+```php
+$connection = new Driver_Phantomjs_Connection;
+$connection->set_phantomjs_binary('_PRJ_/vendor/bin/');
```

Comment:
I think you could use something closer to real-world usage where you would set something like `__DIR__.'/vendor/bin/phantomjs` this way people would see they need to set the binary, not just the path.